src/Layout/Presentation/SingleDistrict/InfoBox.tsx

```diff
@@ -9,38 +9,48 @@ interface InfoBoxProps {
 }
 
 export class InfoBox extends React.Component<InfoBoxProps> {
+    getExtendedInfo(vulnerable: VulnerableDistrictSeat, vulnerableVotes: VulnerableVotes, partyMap: _.Dictionary<string>) {
+        if (vulnerable.runnerUp && vulnerableVotes.partyCode) {
+            if (vulnerable.moreVotesToWin! > vulnerableVotes.moreVotesToWin!) {
+                return <React.Fragment>
+                        &nbsp;
```

Comment:
Why is there a &nbsp; here?

Reply:
Because this fragment will be appended to the end of a sentence. So to keep the last word in the previous sentence and the first in this sentence from merging I add a space.